ext/crypto/shared.rs

```diff
@@ -123,9 +131,9 @@ impl RawKeyData {
       RawKeyData::Public(data) => {
         // public_key is a serialized EncodedPoint
         p384::EncodedPoint::from_bytes(&data)
-          .map_err(|_| type_error("expected valid private EC key"))
+          .map_err(|_| type_error("expected valid public EC key"))
       }
-      _ => Err(type_error("expected private key")),
+      _ => Err(type_error("expected public key data")),
```

Comment:
No support for P384?

If you look at the JWK export code, you can see how we export EC public-keys from private-keys, *both for P256 and P384*, using the PKCS8 decoder in ec_key.rs.

Reply:
Thanks, done